assets/src/edit-story/app/canvas/useCanvasKeys.js

```diff
@@ -252,6 +260,28 @@ function useCanvasKeys(ref) {
   }, [addPastedElements, selectedElements, selectedElementAnimations]);
 
   useGlobalKeyDownEffect('clone', () => cloneHandler(), [cloneHandler]);
+
+  const isPlaying = [
+    STORY_ANIMATION_STATE.PLAYING,
+    STORY_ANIMATION_STATE.PLAYING_SELECTED,
+  ].includes(animationState);
+  useGlobalKeyDownEffect(
+    { key: ['mod+k'] },
+    (evt) => {
+      evt.preventDefault();
+
+      updateAnimationState({
+        animationState: isPlaying
+          ? STORY_ANIMATION_STATE.RESET
+          : STORY_ANIMATION_STATE.PLAYING,
+      });
+
+      trackEvent('canvas_play_animations', {
```

Comment:
Won't this fire on initial render as well?

Reply:
I'm not following. I just adde this here because the same behavior is in the play button callback. does the evt handler fire on initial render for `useGlobalKeyDownEffect`?

Reply:
My bad, somehow thought this was in a `useEffect` 🤦